### DISCUSSION TITLE
Icon request: pedestrian-ramp

### DISCUSSION THREAD

**Discussion by valiyaveedan** [FortAwesome/Font-Awesome#21175] [Icon Requests]
**What can it be used for?**
This icon can be used in case of showing accessibility areas for pedestrians.

**Example image (optional)**
Insert the example image here if necessary.
![image](https://user-images.githubusercontent.com/12865739/63862349-6e959280-c97a-11e9-8d1a-7928017c07ef.png)

**Icon request checklist**
- [x] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x] I have included at least one example image if it could be helpful (optional)
- [x] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by ggireeshbabu** [comment]
+1

**Comment by kshakyaa** [comment]
+1

**Comment by samira-luee** [comment]
+1

**Comment by adubey77** [comment]
+1

**Comment by jonlwc** [comment]
+1

**Comment by cprasuna1711** [comment]
+1

**Comment by prajbav** [comment]
+1

**Comment by anuragnbajpai** [comment]
+1

**Comment by krishthummar** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
